src/components/worker.tsx

```diff
@@ -1,32 +1,33 @@
+import {presets} from "../util/types";
 import {BasicType, CompositeType, isBooleanType, isListType, isVectorType, isBitListType, isBitVectorType, isContainerType, isByteVectorType, isNumberUintType, isBigIntUintType} from '@chainsafe/ssz';
 
-function randomNumber(length: number): number {
```

Comment:
Looks like some of the function input/output types were lost in this file from ts -> js -> ts

Reply:
yeah, forgot to add back the input types.  should all be there now.